code/datums/gamemode/objectives/protect_master.dm

```diff
@@ -0,0 +1,12 @@
+/datum/objective/protect_master
+	var/datum/role/master
+	explanation_text = "Protect and serve your master at all costs."
+
+/datum/objective/protect_master/New(var/datum/role/master)
```

Comment:
objective New() doesn't necessarily have master as its first and only argument. Be wary of this